tensorflow/compiler/mlir/hlo/CMakeLists.txt

```diff
@@ -57,11 +59,16 @@ endif()
 # MLIR/LLVM Configuration
 #-------------------------------------------------------------------------------
 
-find_package(MLIR REQUIRED CONFIG)
-message(STATUS "Using MLIRConfig.cmake in: ${MLIR_DIR}")
-message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
-list(APPEND CMAKE_MODULE_PATH "${MLIR_CMAKE_DIR}")
-list(APPEND CMAKE_MODULE_PATH "${LLVM_CMAKE_DIR}")
+# Skip to find MLIR, which allows to build MHLO as part of another project.
+# This is necessary if the project which embeds MHLO uses a bundled version of
+# MLIR and does not use an installed version of MLIR.
```

Comment:
Wording here is a bit funky

```suggestion
# Find MLIR to install if we are building standalone. If building as part of
# another project, let it handle the MLIR dependency. The dependent project
# might use a bundled version of MLIR instead of installing, for instance.
```

Reply:
Thanks for the suggestion, I have applied it :)

Reply:
Maybe someone can re-add the `ready to pull` label?

Reply:
Apparently I can :-)